src/views/product-integrations-landing/components/faceted-integrations-list/index.tsx

```diff
@@ -1,202 +1,6 @@
-import { IconAward16 } from '@hashicorp/flight-icons/svg-react/award-16'
-import { IconHandshake16 } from '@hashicorp/flight-icons/svg-react/handshake-16'
-import Button from '@hashicorp/react-button'
-import {
-	Integration,
-	IntegrationComponent,
-	Tier,
-} from 'lib/integrations-api-client/integration'
-import React, { useState } from 'react'
-import SearchableIntegrationsList from '../searchable-integrations-list'
+import React from 'react'
 import s from './style.module.css'
 
-interface Props {
-	integrations: Integration[]
-}
-
-export default function FacetedIntegrationList({ integrations }: Props) {
-	// Tier Values
-	const [officialChecked, setOfficialChecked] = useState(false)
-	const [partnerChecked, setPartnerChecked] = useState(false)
-	const [communityChecked, setCommunityChecked] = useState(false)
-
-	// Filter out integrations that don't have releases yet
```

Comment:
The logic has moved to 
[src/views/product-integrations-landing/contexts/integrations-search-context.tsx](https://github.com/hashicorp/dev-portal/pull/1454/files#diff-23b114ae19c9fcb9eb142c6b50c2c5d058c9a4ff657b91ed6274234d8788a33f) and is largely unchanged